README.md

```diff
@@ -3,7 +3,7 @@
 Seal
 ====
 
-Performs sealing of Red Hat Enterprise Linux machine described at
+Performs sealing of Red Hat Enterprise Linux/Centos/Ubuntu/Debian machines as described at
```

Comment:
oVirt site describes sealing process only for RedHat based machines, it doesn't say anything about Debian.

I would leave this sentence as it is, and put new sentence below: In addition it seals Debian based machine ...

... or something like that ... for example put there list ...

This role performs sealing steps for following operating systems

* RedHat / CentOS  ([oVirt site](http://...))
* Debian / Ubuntu

... or other ... ;-)